flink-dist/pom.xml

```diff
@@ -675,6 +675,7 @@ under the License.
 				<artifactId>maven-shade-plugin</artifactId>
 				<executions>
 					<execution>
+						<id>shade-dist</id>
```

Comment:
This is essentially just a safeguard against accidentally inheriting something from somewhere.